backend/sessionManagementRoutes.js

```diff
@@ -111,7 +111,7 @@ const configureRoutes = ({ app, healthApi, tokenRefresher, mailTo, homeLink }) =
   )
   app.get('/autherror', (req, res) => {
     res.status(401)
-    return res.render('pages/autherror', {
+    return res.render('autherror', {
```

Comment:
spotted this copy and paste bug too from notm, auth error page wasn't being rendered